suse_caasp

```diff
@@ -332,23 +332,27 @@ find_and_log_files () {
 }
 
 #############################################################
-section_header "Checking etcd service ..."
+section_header "Checking etcd cluster ..."
 OF=etcd.txt
 plugin_message "All data stored in $OF"
 
-set -a
-[ -f /etc/sysconfig/etcdctl ] && source /etc/sysconfig/etcdctl
-set +a
-if check_rpm etcd && [ -e /usr/bin/etcdctl ]; then
-    rpm_verify $OF etcd
-    log_cmd $OF 'etcdctl --version'
-    log_cmd $OF 'etcdctl member list'
-    log_cmd $OF 'etcdctl cluster-health'
-    log_cmd $OF 'etcdctl get /flannel/network/config'
-    log_cmd $OF 'etcdctl ls /flannel/network/subnets'
-    log_cmd $OF 'systemctl status --full etcd.service'
-    log_cmd $OF 'journalctl -u etcd'
-    conf_files $OF /etc/sysconfig/etcd
```

Comment:
We would like to have output from this command. We have to find a way to run it inside etcd container.

Reply:
I think everything we were doing here in terms of `etcdctl` either is outdated (`flannel`) or we are fetching it with the `curl` commands added, are we missing anything?

Reply:
I think we miss the:

```
    log_cmd $OF 'systemctl status --full etcd.service'
    log_cmd $OF 'journalctl -u etcd'
```

All the others are covered with `curl`, apart from the `flannel` ones -- which are obsolete.

Reply:
How about 
```
ETCD_CNT_ID=$(sudo crictl pods | grep etcd | cut -d" " -f2)
crictl exec -t $ETCD_CNT_ID ETCDCTL_API=3 etcdctl --cacert="/etc/kubernetes/pki/etcd/ca.crt" --cert="/etc/kubernetes/pki/etcd/server.crt" --key="/etc/kubernetes/pki/etcd/server.key" check perf
```

Reply:
Let me give it a try - thanks :)

Reply:
Nope it doesn't work:
```
crictl exec -t $etcdcontainer ETCDCTL_API=3 etcdctl --cacert="/etc/kubernetes/pki/etcd/ca.crt" --cert="/etc/kubernetes/pki/etcd/server.crt" --key="/etc/kubernetes/pki/etcd/server.key" check perf
FATA[0000] execing command in container failed: tty=true must be specified with interactive=true
```

Reply:
I found a way:

```
crictl exec $etcdcontainer sh -c "ETCDCTL_ENDPOINTS='https://127.0.0.1:2379' ETCDCTL_CACERT='/etc/kubernetes/pki/etcd/ca.crt' ETCDCTL_CERT='/etc/kubernetes/pki/etcd/server.crt' ETCDCTL_KEY='/etc/kubernetes/pki/etcd/server.key' ETCDCTL_API=3 etcdctl check perf"
````